pkg/util/build/build.go

```diff
@@ -20,4 +21,5 @@ func init() {
 	version.Branch = Branch
 	version.BuildUser = BuildUser
 	version.BuildDate = BuildDate
+	version.GoVersion = "???"
```

Comment:
How can I access the Go version?

Reply:
You can use the `runtime` package: https://golang.org/pkg/runtime/#Version

Reply:
That's how Prometheus does it as well: https://github.com/prometheus/common/blob/main/version/info.go#L33